examples/reinforcement learning/env_checker.py

```diff
@@ -0,0 +1,71 @@
+# Basic imports
+import gymnasium
+import numpy as np
+import os
+from gymnasium.wrappers import FlattenObservation
+
+# Stable baselines 3
+''' 
+Stable baselines 3 has a build-in function called `check_env` that checks if the environment is compatible with the library. 
+It checks the following:
+    - Observation space
+    - Action space
+    - Reward range
+    - Whether the environment is vectorized or not
+    - Whether the environment uses a `Dict` or `Tuple` observation space
+    - Whether the environment uses a `Dict` or `Tuple` action space
+    - Whether the environment uses a `VecEnv` or not
+    - Whether the environment uses a `VecNormalize` wrapper or not
+    - Whether the environment uses a `FlattenObservation` wrapper or not
+    - Whether the environment uses a `FrameStack` wrapper or not
+    - Whether the environment uses a `TimeLimit` wrapper or not
+    - Whether the environment uses a `Monitor` wrapper or not
+    - Whether the environment uses a `VecFrameStack` wrapper or not
+    - Whether the environment uses a `VecTransposeImage` wrapper or not
+'''
+from stable_baselines3.common.env_checker import check_env
```

Comment:
This introduces a dependency on stable baselines. However, RL is one use case and urdfenvs should done add dependencies that are specific do a use case.

I think the whole folder should not be in here.

Reply:
But I think it's ok to have `SB3Float32ActionWrapper`, don't you think? It does not add any dependency and uses gymnasium.